actions/loadconfig.py

```diff
@@ -16,12 +16,14 @@ def run(self, config_file, method, **std_kwargs):
                                   'merge or replace').format(method))
 
         with self.get_driver(**std_kwargs) as device:
+            # inline_transfer: If set it becomes True, else False
+            device.inline_transfer = inline_transfer
```

Comment:
I reviewed the `inline_transfer` implementation in the NAPALM IOS driver, and this almost seems like it would be better to implement this in the config, and pass in via `optional_args`. Doing this would re-use the built-in logic in the driver for deciding what the default value for this parameter should be - assigning it on the instance like this bypasses that logic in the driver's `__init__` function. Thoughts?

Reply:
Part of me agrees and part of me disagrees. 

Agree: Getting the `__init__()` function to do this from the config (per host entry) would be tidy. It did cross my mind originally but decided against it. 

Disagree: I've had mixed experience with different versions of OS that the ios driver communicates with. If it's an action decision as opposed to config, then if the action fails, we can  try again on error transition to try inline as opposed to SCP. 

Right now I'm leaning 60/40 in favour of action over config. Happy to take your lead on this.

Thoughts?

Reply:
If that's true, then the inconsistency should *definitely* get fixed at the NAPALM layer. Definitely open an issue with them if you're seeing differences in OS behavior.

Since the other drivers effectively ignore this for now, we can leave it in the action - but if the underlying behavior changes, or if they change how the default gets set, this may have to move.